next.config.js

```diff
@@ -60,7 +62,7 @@ if (process.env.ANALYZE === 'true') {
     const withBundleAnalyzer = require('@next/bundle-analyzer')({
         enabled: true,
     });
-    module.exports = withBundleAnalyzer(configVals);
+    module.exports = withNextPluginPreval(withBundleAnalyzer(configVals));
 } else {
-    module.exports = configVals;
+    module.exports = withNextPluginPreval(configVals);
```

Comment:
The preval data will be re-fetched at build time (every hour or so).

Reply:
what is preval data

Reply:
It is static data that is used across multiple pages. Rather than requesting the data via Strapi each time, I am using this plugin that will prevent the additional API calls and just use static data retrieved during initial build. https://github.com/ricokahler/next-plugin-preval#why.

```
Site-wide data: if you have static data that's required across many different pages, getStaticProps is a somewhat awkward mechanism because for each new page, you'll have to re-fetch that same static data. For example, if you use getStaticProps to fetch content for your header, that data will be re-fetched on every page change.
```